src/recipe/Baked.Recipe.Service.Application/Localization/AspNetCore/AspNetCoreLocalizationFeature.cs

```diff
@@ -14,6 +15,51 @@ public class AspNetCoreLocalizationFeature(CultureInfo _language,
 {
     public void Configure(LayerConfigurator configurator)
     {
+        configurator.ConfigureGeneratedFileCollection(files =>
+        {
+            if (configurator.IsNfr())
+            {
+                return;
+            }
+
+            var localeDir = Path.Combine(Assembly.GetEntryAssembly()?.Location ?? throw new("'EntryAssembly' shoul have existed"), "../../../../Locales");
+
+            configurator.UsingLocaleDictionary(locales =>
+            {
+                files.AddAsJson(FillLocales(_language, localeDir, locales, defaultLanguage: true), name: $"locale.{_language.Name}", outdir: "Ui");
+
+                if (_otherLanguages is not null)
+                {
+                    foreach (var language in _otherLanguages)
+                    {
+                        files.AddAsJson(FillLocales(language, localeDir, locales), name: $"locale.{language.Name}", outdir: "Ui");
+                    }
+                }
+            });
+
+            Dictionary<string, string> FillLocales(CultureInfo language, string resourceDir, ILocaleDictionary locales,
+                bool defaultLanguage = false
+            )
+            {
+                var result = new Dictionary<string, string>(locales);
+
+                var resourceFilePath = Path.Combine(resourceDir, defaultLanguage ? $"locale.restext" : $"locale.{language.Name}.restext");
+                if (File.Exists(resourceFilePath))
+                {
+                    var values = AspNetCoreLocalizationExtensions.ReadResxFileAsDictionary(resourceFilePath);
+                    foreach (var (key, value) in values)
+                    {
+                        if (result.ContainsKey(key))
+                        {
+                            result[key] = value;
+                        }
+                    }
+                }
+
+                return result;
+            }
```

Comment:
this and `ReadResxFileAsDictionary` can be extracted to a separate class. i think it would be a better choice.